frontend/src/app/register/register.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { RegisterRequestModel } from 'app/auth.models';
+import { AuthService } from '../auth.service';
+import { Observable } from 'rxjs/Observable';
+import { RedirectComponent } from 'app/redirect.component';
+import { Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-register',
+  templateUrl: './register.component.html',
+})
+export class RegisterComponent implements OnInit {
+  registerModel: RegisterRequestModel = {
+    email: '',
+    password: '',
+  };
+  ACCEPT_TERMS = 'Wyrażam zgodę na przetwarzanie moich danych osobowych';
+  registrationSuccessful = false;
+  userIsAuthenticated = false;
+
+  constructor(private authService: AuthService,
+  ) {
+  }
+
+  register(): void {
+    this.registrationSuccessful = false;
+    this.userIsAuthenticated = false;
+    this.authService.register(this.registerModel.email, this.registerModel.password)
+      .subscribe(rsp => {
```

Comment:
For now I'm fine with that solution (we will refine it later) but generally we should hide all http artifacts inside service method, so there shouldn't be `201` or `400` available here, there should be only some kind of abstract `success` and `failure`